summarization/extractor_output_preprocessor.py

```diff
@@ -82,12 +90,13 @@ def get_preprocessed_content_lists(self):
         # set title text objects
         self._set_sentence_objects_list_for_title_sentences()
 
-        return {
-            "titles": self.title_text_objects_list,
-            "sentences": self.sentence_objects_list,
-            "media": self.media_content_list,
-            "embedded_content": self.embedded_content_list
-        }
+        return PreprocessedContents(
```

Comment:
Change method name
`get_preprocessed_contents` should be fine.
It doesn't return lists anymore.

Reply:
updated accordingly.